### PR TITLE
Support echo instrument

### PR DESCRIPTION
**Pull request by ethan256**
support [echo](github.com/labstack/echo/v4) instrumentation

### CODE REVIEW

**Review by mrproliu — CHANGES_REQUESTED** [review]
Please fix the comments and update the [plugin documentation](https://github.com/apache/skywalking-go/blob/main/docs/en/agent/support-plugins.md) and [CHANGES.md](https://github.com/apache/skywalking-go/blob/main/CHANGES.md).

File: go.work [hunk 1]
```diff
@@ -6,16 +6,17 @@ use (
 	// define the plugins
 	./plugins/core
 	./plugins/dubbo
+	./plugins/echo
```
Comment (mrproliu) [inline go.work:9]: Why did you change to the `echo` otherwise `echov4`?

File: plugins/echov4/interceptor.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+		return func(c echo.Context) error {
+			request := c.Request()
+			span, err := tracing.CreateEntrySpan(
+				fmt.Sprintf("%s:%s", request.Method, c.Path()), func(headerKey string) (string, error) {
+					return request.Header.Get(headerKey), nil
+				},
+				tracing.WithLayer(tracing.SpanLayerHTTP),
+				tracing.WithTag(tracing.TagHTTPMethod, request.Method),
+				tracing.WithTag(tracing.TagURL, request.Host+request.URL.Path),
+				tracing.WithComponent(5006))
```
[the remark this answers is not in the document]
Comment (mrproliu) [reply plugins/echov4/interceptor.go]: The PR has been merged. Please keep fixing other issues.

**Comment by wu-sheng** [comment]
Could you provide a correct link to echo framework? Is it a rpc or something?

**Comment by wu-sheng** [comment]
It is like a MIT licensed lib, do you need an icon for it on the topo? If so, please send a PR for that as well, otherwise, by default, it would be an earth icon only for server side.

**Comment by wu-sheng** [comment]
@ethan256 Any update?

**Comment by wu-sheng** [comment]
You are instrumenting a server lib, so generally, you should add the project logo to our website.

**Comment by mrproliu** [comment]
Also, please resolve the conflict and conversations.

**Comment by wu-sheng** [comment]
Close for now, it seems contributor has no interest to follow reviewers comments.
